packages/hub/.eslintignore

```diff
@@ -0,0 +1,11 @@
+/node_modules/
+/coverage/
+!.*
+/.vscode/
+/typings
+/migrations/
+**/*.json
+/website/
+/.etherlime-store/
+/lib/
+/dist/
```

Comment:
Nit, should this contain a newline at the end of the file?

Reply:
I am on Windows 😬
Should we enforce line endings?

Reply:
Ah gotcha. It might be nice to enforce a newline at the end of files.